.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,3 +16,20 @@ Bullet points like
 really help with making this more readable.
 
 Fixes/Closes/Related-to #1 (enter issue number, except in rare cases where none exists).
+
+It is also helpful to add an emoji before the commit message to identify the kind of work done on a single commit. See the following suggestions below:
```

Comment:
Is the idea to select one emoji? Or all that are applicable?

Reply:
well ideally you should be doing one thing with commits (of course i understand in practice this doesn't happen), so either way.